packages/components/src/__future__/Tag/RemoveButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ButtonHTMLAttributes } from "react"
+import classnames from "classnames"
+import { CloseIcon } from "~components/Icon"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./RemoveButton.module.scss"
+
+export type RemoveButtonProps = OverrideClassName<
```

Comment:
https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3324903621/RemovableTag#API

The props outlined in the doc should be put here, and the prop within the Tag should reference these props.